File: src/com/brianc/layout/EdgeSizes.java
package com.brianc.layout;

public class EdgeSizes {
	public float left;
	public float right;
	public float top;
	public float bottom;

	public EdgeSizes(float left, float right, float top, float bottom) {
		this.left = left;
		this.right = right;
		this.top = top;
		this.bottom = bottom;
	}
	
	public EdgeSizes() {
	}
	
	public EdgeSizes(EdgeSizes sizes) {
		this(sizes.left, sizes.right, sizes.top, sizes.bottom);
	}
}

File: src/com/brianc/layout/EdgeSizes.kt
package com.brianc.layout

data class EdgeSizes(var left: Float = 0f, var right: Float = 0f, var top: Float = 0f, var bottom: Float = 0f) {
    constructor(sizes: EdgeSizes) : this(sizes.left, sizes.right, sizes.top, sizes.bottom)
}
